x-pack/plugins/fleet/server/services/epm/packages/install.ts

```diff
@@ -488,7 +489,6 @@ export async function installPackage(args: InstallPackageParams) {
     return response;
   } else if (args.installSource === 'upload') {
     const { archiveBuffer, contentType, spaceId } = args;
-    logger.debug(`kicking off install of uploaded package`);
```

Comment:
I removed this because it added some noise in logs during setup, as the bundled installation process uses this method. Since the upload feature is largely unused anyway, I figured we didn't need this log right now. Happy to replace it if desired.